sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+because the BTC is safe if at least 30% of the stackers are honest -- they can 
+deny 69.999...% of dishonest Stackers the ability to abscond with the BTC.  Thus, this
+proposal calls for enabling Stackers to set the capital limit themselves, as
+long as the limit has at least 70% vote support.  To help build user trust
+in the safety of the BTC,  we expect Stackers to disclose additional information
+about themselves (e.g. business name, mailing address, corporate charter,
+portion of STX owned, etc.) in order to provide convincing evidence that this
+30% honest coalition exists.  If this 30% honest coalition exist, then there is
+no ROI for dishonest Stackers to participate if their goal is to steal the BTC.
+
+## Future Work: Recovery Mode
+
+This SIP proposes implementing the aforementioned Normal mode for sBTC.  A
+follow-up SIP may propose a Recovery mode, whereby instead of burning BTC while
+the system is frozen, the BTC is instead repurposed to directly fulfill pending
+withdraw requests.  While withdraw fulfillment will be considerably slower, at
+least the sBTC users will eventually be made whole in the event of a
+catastrophe.
+
+In Recovery mode, sBTC deposits would cease and all pending sBTC deposits would be
+cancelled.  Withdraws would be fulfilled in first-in first-out order from
+redirected PoX payouts.  Once a withdraw request has been
+satisfied -- i.e. the requester's BTC address has
+received _at least_ as many satoshis as reqeusted from the redirected PoX
+payouts -- then the withdraw request completes and the next requester's withdraw
+is processed.
+
+Once the Stacks blockchain supports Recovery Mode, the system would continue
+operating even if the system was frozen for many reward cycles.  The blockchain
+would not halt; instead, it would continue its operation to fulfill all withdraw
+requests while the BTC is seemingly inaccessible to Stackers.  PoX payouts and
+STX unlocks would only resume once the current set of Stackers control
+enough BTC to back each sBTC token, and are able to fulfill pending sBTC
+withdraws.
+
+The system would transition back to Normal mode once all pending withdraw
+transactions have been fulfilled, and once the current set of Stackers possess
+enough BTC to cover all outstanding sBTC tokens.
+
+# Related Work
+
+There are several other systems which implement a wrapped BTC asset on a
+different chain.  The key difference between most of them and sBTC is that the
+nodes which control the BTC wallet use a closed-membership protocol to decide
+which parties can enter and leave the set.  In detail:
+
+## wBTC 
+
+This is a closed membership system.  It is made up of
+50+ merchants and custodians with keys to the wBTC
+multi-sig contract on Ethereum. End users purchase wBTC directly from
+authorized merchants, and often the experience is fast.  However, wBTC deposits and
+withdrawals can only be performed by authorized merchants and custodians. 
```

Comment:
```suggestion
## WBTC 

This is a closed membership system.  It is made up of
50+ merchants and custodians with keys to the WBTC
multi-sig contract on Ethereum. End users purchase WBTC directly from
authorized merchants, and often the experience is fast.  However, WBTC deposits and
withdrawals can only be performed by authorized merchants and custodians. 
```

It is stylized `WBTC` as seen on their website: https://wbtc.network/

Might be worth noting that although the merchants manage issuance and redemption, all BTC backing WBTC is held by a single company (BitGo).

Reply:
cc @andreserrano